src/components/list/list-item.view.js

```diff
@@ -8,7 +8,7 @@ const ListItem = Backbone.View.extend(
          * @member {String} className - the class attribute of the element
```

Comment:
list-item.html is missed.

Reply:
list-item.html is located in src/components/list/list-item.html. I do not observe any problems with this.